target/linux/ramips/image/mt7620.mk

```diff
@@ -362,6 +362,20 @@ define Device/elecom_wrh-300cr
 endef
 TARGET_DEVICES += elecom_wrh-300cr
 
+define Device/fon_fon2601
+  MTK_SOC := mt7620a
+  IMAGE_SIZE := 16318464
```

Comment:
Please use `15936k` here for consistency.

Reply:
modified.